v21.2/scalar-expressions.md

```diff
@@ -27,30 +26,28 @@ They are described further in section [SQL Constants](sql-constants.html).
 
 An expression in a query can refer to columns in the current data source in two ways:
 
-- Using the name of the column, e.g., `price` in `SELECT price FROM
+- The name of the column, e.g., `price` in `SELECT price FROM
   items`.
 
   - If the name of a column is also a
   [SQL keyword](keywords-and-identifiers.html#keywords), the name
-  must be appropriately quoted. For example: `SELECT "Default" FROM
-  configuration`.
+  must be appropriately quoted. For example: `SELECT "Default" FROM configuration`.
 
   - If the name is ambiguous (e.g., when joining across multiple
   tables), it is possible to disambiguate by prefixing the column
   name by the table name. For example, `SELECT items.price FROM
   items`.
 
-- Using the ordinal position of the column. For example, `SELECT @1
-  FROM items` selects the first column in `items`.
+- The ordinal position of the column. For example, `SELECT @1 FROM items` selects the first column in `items`.
 
-  *This is a CockroachDB SQL extension.*
+    {{site.data.alerts.callout_info}}This is a CockroachDB SQL extension.{{site.data.alerts.end}}
 
-  {{site.data.alerts.callout_danger}}
-  Ordinal references should be used with care in production
-  code! During schema updates, column ordinal positions can change and
-  invalidate existing queries that use ordinal positions based on a
-  previous version of the schema.
-  {{site.data.alerts.end}}
+    {{site.data.alerts.callout_danger}}
```

Comment:
nit: the line breaks in this section are weird. Actually, the line breaks within the page could all be cleaned up.